les-3-opdracht-grid/styles.css

```diff
@@ -0,0 +1,53 @@
+.grid-container {
+    background-color: burlywood;
```

Comment:
Voor snelle toepassingen is een kleurnaam handig. Wist je dat je deze met de alt+enter simpel om kan zetten naar een ander type?